src/Layout/redist/trustedroots/README.md

```diff
@@ -0,0 +1,43 @@
+# Trusted X.509 Certificate Roots
+
+This directory contains [Microsoft-maintained root certificate trust lists (CTL)](https://learn.microsoft.com/security/trusted-root/program-requirements) for [Code Signing (CS)](https://en.wikipedia.org/wiki/Code_signing), and [Time Stamping (TS)](https://en.wikipedia.org/wiki/Code_signing#Time-stamping). The .NET SDK uses these CTLs for [NuGet signing and validation](https://learn.microsoft.com/en-us/nuget/reference/signed-packages-reference), on Linux and macOS. They are also used in various Microsoft products. 
+
+- [Code signing CTL](codesignctl.pem)
+- [Timestamping CTL](timestampctl.pem)
+
+The CTLs are stored in [PEM format](https://en.wikipedia.org/wiki/Privacy-Enhanced_Mail). 
+
+## Behavior
+
+[NuGet uses these CTLs as a fallback](https://github.com/dotnet/sdk/issues/25686) when an OS-provided CTL is not available.
+
+- Linux: The fallback CTL is typically used, however, an OS-provided CTL will be used if available (see `ca-certificates` section).
+- macOS: The fallback CTL is always used since an OS-provided CTL is never available, when signed package verification is enabled (which is not the default).
```

Comment:
```suggestion
- macOS: The fallback CTL is always used when signed package verification is enabled, which is not the default since an OS-provided CTL is never available.
```